hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/nssummary/NSSummaryAdmin.java

```diff
@@ -115,6 +115,11 @@ public boolean isNotValidBucketOrOBSBucket(String path) {
         OzoneConfiguration.of(getOzoneConfig()));
     try (OzoneClient ozoneClient = OzoneClientFactory.getRpcClient(getOzoneConfig())) {
       ObjectStore objectStore = ozoneClient.getObjectStore();
+      // Return false if path is root, just a volume
```

Comment:
Pls correct and update the comment here as well.

Reply:
@devmadhuu may I know what was wrong with this comment. I meant to say that we are returning false if path is root i.e., `ofsPath.getVolumeName().isEmpty()` or if it's just volume i.e., `ofsPath.getBucketName().isEmpty()`

Reply:
Comment is confusing. I meant, if path is root, and no volume or bucket name is present in path, then what we are trying to say in comment with "`just a volume`" , just "`Return false if path is root`" should have been enough. Am i missing something here ?

Reply:
@devmadhuu We are returning a false with the path is root like `/` or path has just the volume and no bucket like `/volume`